### PR TITLE
feat: i18n support for Greek language

### PR DESCRIPTION
**Pull request by IamsahilPhadtare**
Added Greek Language Support

### CODE REVIEW

File: docs/docs/notification-center/react-components.md [hunk 1]
```diff
@@ -197,6 +197,7 @@ The `i18n` prop can accept 2 different types of values
           <li><code>pu</code> (Punjabi)</li>
           <li><code>ta</code> (Tamil)</li>
           <li><code>ko</code> (Korean)</li>
+          <li><code>gk</code> (Greek)</li>
```
Comment (p-fernandez) [inline docs/docs/notification-center/react-components.md:200]: Greek language ISO code is `el`.

File: packages/notification-center/src/i18n/languages/gk.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { ITranslationEntry } from '../lang';
+
+export const GK: ITranslationEntry = {
+  translations: {
+    notifications: 'ειδοποιήσεις',
+    markAllAsRead: 'επισημάνετε όλα ως αναγνωσμένα',
+    poweredBy: 'τροφοδοτείται από',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/gk.ts:7]: Missing the `settings` translation.

**Review by p-fernandez — CHANGES_REQUESTED** [review]

File: packages/notification-center/src/i18n/languages/gk.ts (hunk 2 above)
Comment (IamsahilPhadtare) [reply packages/notification-center/src/i18n/languages/gk.ts]: resolved

**Review by IamsahilPhadtare — COMMENTED** [review]
Lang Abbreviation Changed and Settings Translated

**Pull request merged**